src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
`body` doesn't have default `padding`, delete this

Reply:
fixed